src/data/fintype/basic.lean

```diff
@@ -483,16 +486,27 @@ arbitrary `fintype` instances, use either `fintype.card_le_one_iff_subsingleton`
   fintype.card α = 1 :=
 subsingleton.elim (of_subsingleton $ default α) h ▸ card_of_subsingleton _
 
+@[priority 100] -- see Note [lower instance priority]
+instance of_is_empty [is_empty α] : fintype α := ⟨∅, is_empty_elim⟩
+
+/-- Note: this lemma is specifically about `fintype.of_is_empty`. For a statement about
+arbitrary `fintype` instances, use `fintype.univ_eq_empty'`. -/
```

Comment:
Is this `'` deliberate? I don't see the primed lemma in doc-gen.

Reply:
added in L54 this PR :)